c2corg_ui/static/js/search.js

```diff
@@ -196,8 +196,8 @@ app.SearchController.prototype.createDataset_ = function(type) {
       header: (function() {
         return '<div class="header" dataset="' + type + '">' + this.gettextCatalog_.getString(type) + '</div>';
       }).bind(this),
-      footer: function() {
-        return '<p class="suggestion-more"><a href="/' + type + '" class="green-text">+ see more results</a></p>';
+      footer: function(doc) {
+        return '<p class="suggestion-more"><a href="/' + type + '?keyword=' + doc['query'] + '" class="green-text">+ see more results</a></p>';
```

Comment:
The URL will rather be something like
```
'/' + type + '/keyword/' + doc['query']
```
As in v5 when you type "grange" in the simple search field at the top of the pages => it gives a URL such as http://www.camptocamp.org/summits/list/snam/grange/orderby/snam/order/asc

In addition you probably have to URL-encode ``doc['query']``.